redbot/cogs/customcom/customcom.py

```diff
@@ -204,15 +204,16 @@ def __init__(self, bot):
         """Custom commands management."""
         pass
 
-    @customcom.group(name="create", aliases=["add"])
+    @customcom.group(name="create", aliases=["add"], invoke_without_command=True)
     @checks.mod_or_permissions(administrator=True)
-    async def cc_create(self, ctx: commands.Context):
+    async def cc_create(self, ctx: commands.Context, command: str.lower, *, text: str):
         """Create custom commands.
 
+        If a type is not specified, a simple CC will be created.
         CCs can be enhanced with arguments, see the guide
         [here](https://red-discordbot.readthedocs.io/en/v3-develop/cog_customcom.html).
         """
-        pass
+        await ctx.invoke(self.cc_create_simple, command=command, text=text)
```

Comment:
`ctx.invoke` doesn't check permissions, but it's probably fine here specifically...

Reply:
I believe the group command has the same permissions as the subcommand, so that check should be fine.

Reply:
Technically speaking, because permissions can be customized this is never a safe assumption. The only case here where this would be problematic is if non-simple ccs were allowed, while simple ones were denied though.